datafusion/functions-aggregate/src/approx_distinct.rs

```diff
@@ -126,6 +126,28 @@ where
     }
 }
 
+#[derive(Debug)]
+struct StringViewHLLAccumulator<T>
+where
+    T: OffsetSizeTrait,
+{
+    hll: HyperLogLog<String>,
+    phantom_data: PhantomData<T>,
+}
+
+impl<T> StringViewHLLAccumulator<T>
+where
+    T: OffsetSizeTrait,
+{
+    /// new approx_distinct accumulator
```

Comment:
I understand you follow the previous pattern, but the comment seems unclear vague
```suggestion
```

Reply:
Good point @Weijun-H , addressed in latest PR.